### PR TITLE
Add shift_assignments documentation

### PR DESCRIPTION
**Pull request by mischasteiner73**

### CODE REVIEW

File: shift_assignments.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+| `id`                  | `number` | no           | ID                                                                      | `1`                                   |
+| `date`                | `string` | no           | Date of the assignment in format `yyyy-mm-dd`                           | `2020-06-06`                          |
+| `position_id`         | `number` | yes          | ID of the employee                                                      | `232`                                 |
+| `created_at`          | `string` | no           | Datetime in format `yyyy-mm-dd HH:MM:SS.XXXXXXXXX +0000`                | `2021-04-19 12:00:00.000000000 +0000` |
+| `updated_at`          | `string` | no           | Datetime in format `yyyy-mm-dd HH:MM:SS.XXXXXXXXX +0000`                | `2021-04-19 12:00:00.000000000 +0000` |
+| `posted_on_market_on` | `string` | yes          | Date of the day when shift_assignment was posted on market `yyyy-mm-dd` | `2020-06-22`                          |
+| `unique_aabrv`        | `string` | yes          | A unique abbreviation of the shift_assignment                           | `NS-a`                                |
```
Comment (lxxxvi) [inline shift_assignments.md:92]:
```suggestion
| `unique_abbrv`        | `string` | yes          | A unique abbreviation of the shift_assignment                           | `NS-a`                                |
```

File: shift_assignments.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+
+## Shifttype
+
+| Key                         | Type     | Can be null? | Description                                                    | Example values                        |
+| --------------------------- | -------- | ------------ | -------------------------------------------------------------- | ------------------------------------- |
+| `id`                        | `number` | no           | ID                                                             | `1`                                   |
+| `name`                      | `string` | yes          | Name                                                           | `Nachtschicht`                        |
+| `starttime`                 | `string` | no           | Starttime of the shifttype in format `HH:MM`                   | `17:15`                               |
+| `endtime`                   | `string` | no           | Endtime of the shifttype in format `HH:MM`                     | `01:00`                               |
+| `responsability_id`         | `number` | no           | ID of the responsability                                       | `121`                                 |
```
Comment (lxxxvi) [inline shift_assignments.md:103]: I'd not reveal any foreign keys if we don't have to.

File: shift_assignments.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+| `remark`                    | `string` | yes          | The remark                                                     | `Mit abrechnen`                       |
+| `prefix`                    | `string` | yes          | The abbreviation                                               | `NS`                                  |
+| `color`                     | `string` | yes          | The color                                                      | `#105020`                             |
+| `plannable_category_id`     | `number` | no           | Id of the grouping                                             | `857`                                 |
```
Comment (lxxxvi) [inline shift_assignments.md:107]: Same here, can we avoid to expose this ?

File: shift_assignments.md [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+#### Example
+
+```
+curl GET https://app.pia-planer.ch/api/v1/shift_assignments?from=2021-05-01&to=2021-05-31&state=released \
+   -H 'Authorization: Business YOUR_API_KEY'
+```
+
+### Response
+
+Response is an `Object`.
```
Comment (lxxxvi) [inline shift_assignments.md:36]:
```suggestion
Response is an `Array`.
```

File: shift_assignments.md [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+
+##### Labels
+
+```json
+[
+  {
+    "name": "Essen, CHF 14",
+    "technical_name": "lunch"
+  }
+]
```
Comment (lxxxvi) [inline shift_assignments.md:125]: I think we can remove the surrounding `Array` here.

File: shift_assignments.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
...
+##### Pauses
+
+```json
+[
+  {
+    "id": 288754210,
+    "starttime": "22:00",
+    "endtime": "22:15"
+  }
+]
```
Comment (lxxxvi) [inline shift_assignments.md:142]: I think we can remove the surrounding `Array` here

File: shift_assignments.md [hunk 7]
```diff
@@ -0,0 +1,149 @@
...
+      "working_duration": 9.75,
+      "working_duration_in_hh_mm": "09:45",
+      "created_at": "2023-09-18T13:24:34.605Z",
+      "updated_at": "2023-09-18T13:24:34.605Z"
+    }
+  }
+]
+```
+
+## Item attributes
```
Comment (lxxxvi) [inline shift_assignments.md:82]: Let's add quotes to the strings in the example values.

File: shift_assignments.md [hunk 8]
```diff
@@ -0,0 +1,149 @@
...
+/api/v1/shift_assignments
+```
+
+#### Params
+
+| Param   | Required | Description                               |
+| ------- | -------- | ----------------------------------------- |
+| `from`  | true     | Startdate in format ISO 8601 `2021-05-01` |
+| `to`    | true     | Enddate in format ISO 8601 `2021-05-31`   |
+| `state` | false    | Possible values: `released`, `all`        |
```
Comment (lxxxvi) [inline shift_assignments.md:25]: Technically there is no such thing like `all`... let's discuss in the app's PR.

Comment (lxxxvi) [reply shift_assignments.md:25]: Also, I think we should explain what `released` means here.

**Review by lxxxvi — APPROVED** [review]
Please also see the comments in the app's PR.

File: shift_assignments.md (hunk 6 above)
Comment (mischasteiner73) [reply shift_assignments.md:140]: Hmm, I even think we have to change the description to "Response is an array...."

File: shift_assignments.md (hunk 5 above)
Comment (mischasteiner73) [reply shift_assignments.md:123]: Hmm, but we also did it like this in working_hours...

**Review by lxxxvi — APPROVED** [review]

**Pull request merged**